mmk_default_rules_alpha.mms

```diff
@@ -195,6 +195,47 @@ GENCAT	    = GENCAT
 GENCATFLAGS =
 
 
+!SDL_UNSUPPORTED_LANGUAGES=DTR:.? DUMP:.? UIL:.UIL?
+!SDL_LANGUAGES=ADA:.ADA BASIC:.BAS BLISS:.R32 BLISS64:.R64 CC:.C DCL:.COM -
+!	FORTRAN:.FOR MACRO:.MAR PASCAL:.PAS PLI:.PL1 SDML:.SDML -
+! 	TPU:.TPU
+!$(FOREACH LANG,$(SDL_LANGUAGES),-
+!.SDL$(LASTWORD $(PATSUBST :, ,$(LANG))) : ; -
+!$(SDL) $(SDLFLAGS)/LANG=$(FIRSTWORD $(PATSUBST :, ,$(LANG)))=$(MMS$TARGET) $(MMS$SOURCE))
+.SDL.ADA :
+    $(SDL)$(SDLFLAGS)/LANG=ADA=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.BAS :
+    $(SDL)$(SDLFLAGS)/LANG=BASIC=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.R32 :
+    $(SDL)$(SDLFLAGS)/LANG=BLISS=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.R64 :
+    $(SDL)$(SDLFLAGS)/LANG=BLISS64=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.C :
+    $(SDL)$(SDLFLAGS)/LANG=CC=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.COM :
+    $(SDL)$(SDLFLAGS)/LANG=DCL=$(MMS$TARGET) $(MMS$SOURCE)
+!.SDL.? :
+!    $(SDL)$(SDLFLAGS)/LANG=DTR=$(MMS$TARGET) $(MMS$SOURCE)
+!.SDL.? :
+!    $(SDL)$(SDLFLAGS)/LANG=DUMP=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.FOR :
+    $(SDL)$(SDLFLAGS)/LANG=FORTRAN=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.MAR :
+    $(SDL)$(SDLFLAGS)/LANG=MACRO=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.PAS :
+    $(SDL)$(SDLFLAGS)/LANG=PASCAL=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.PL1 :
+    $(SDL)$(SDLFLAGS)/LANG=PLI=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.SDML :
+    $(SDL)$(SDLFLAGS)/LANG=SDML=$(MMS$TARGET) $(MMS$SOURCE)
+.SDL.TPU :
+    $(SDL)$(SDLFLAGS)/LANG=TPU=$(MMS$TARGET) $(MMS$SOURCE)
+!.SDL.UIL? :
+!    $(SDL)$(SDLFLAGS)/LANG=UIL=$(MMS$TARGET) $(MMS$SOURCE)
+SDL	    = SDL/ALPHA
+SDLFLAGS    = /C_DEVELOPMENT/VMS_DEVELOPMENT
```

Comment:
It is a tricky one.  I was thinking of defining SDL as:
```
SDL = SDL/$(MMSARCH_NAME)
```
However, the problem there is that it doesn't work for Itanium (or x86-64).  So, it would need some kinds of $(IF ...) test.  Then, /C_DEVELOPMENT and /VMS_DEVELOPMENT do different things too.